march_rqt_gait_generator/src/march_rqt_gait_generator/model/GaitGeneratorSetpoint.py

```diff
@@ -0,0 +1,6 @@
+from march_shared_classes.gait.Setpoint import Setpoint
+
+class GaitGeneratorSetpoint(Setpoint):
+    def invert(self, duration):
```

Comment:
Because it is only useful for the gait generator. Gaits should only be editted in the gait generator, and hence, functions that edit gaits should only be present there.